packages/dds/tree/src/tableSchema.ts

```diff
@@ -595,22 +591,21 @@ export namespace System_TableSchema {
 				return rows as unknown as RowValueType[];
 			}
 
-			public setCell({ key, cell }: TableSchema.SetCellParameters<TCellSchema>): void {
```

Comment:
This was a type bug, and was the real source of the need for type casting below.